### PR TITLE
 more API review  changes

### PR DESCRIPTION
**Pull request by hemanttanwar**
Based on review comment on : https://apiview.azurewebsites.net/Assemblies/Review/2a23558e091248958cbfd49125e58cb7
1. Change Uri -> Url
2. rename function name 
3. Use Duration instead of long.

### CODE REVIEW

File: sdk/identity/azure-identity/src/main/java/com/azure/identity/credential/AadCredentialBuilderBase.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
     /**
      * Specifies the Azure Active Directory endpoint to acquire tokens.
-     * @param authorityHost the Azure Active Directory endpoint
+     * @param authorityHost the Azure Active Directory endpredirectUrioint
```
Comment (jianghaolu) [inline sdk/identity/azure-identity/src/main/java/com/azure/identity/credential/AadCredentialBuilderBase.java]: typo?

File: sdk/identity/azure-identity/src/main/java/com/azure/identity/credential/AuthorizationCodeCredentialBuilder.java [hunk 2]
```diff
@@ -39,11 +39,11 @@ public AuthorizationCodeCredentialBuilder authorizationCode(String authCode) {
      * to complete the {@link AuthorizationCodeCredential} construction from there.
      * This is also called Reply URLs in some contexts.
      *
-     * @param redirectUri the redirect URI to send the authorization code
+     * @param redirectUrl the redirect URL to send the authorization code
```
Comment (jianghaolu) [inline sdk/identity/azure-identity/src/main/java/com/azure/identity/credential/AuthorizationCodeCredentialBuilder.java]: The issue with this change is that this does not match the name in other places / libraries Microsoft publishes e.g. https://github.com/AzureAD/microsoft-authentication-library-for-java/blob/dev/src/main/java/com/microsoft/aad/msal4j/AuthorizationCodeParameters.java#L41.

@JonathanGiles

Comment (JonathanGiles) [reply sdk/identity/azure-identity/src/main/java/com/azure/identity/credential/AuthorizationCodeCredentialBuilder.java]: From the reviews we did 'offline', I still lean towards URL.

**Comment by abagarwams** [comment]
Not sure what this is about?

File: sdk/identity/azure-identity/src/main/java/com/azure/identity/DeviceCodeChallenge.java [hunk 3]
```diff
@@ -18,31 +18,27 @@
      *
      * @param userCode code which user needs to provide when authenticating at the verification URI
      * @param deviceCode code which should be included in the request for the access token
-     * @param verificationUri URI where user can authenticate
+     * @param verificationUrl URI where user can authenticate
```
Comment (conniey) [inline sdk/identity/azure-identity/src/main/java/com/azure/identity/DeviceCodeChallenge.java]:
```suggestion
     * @param verificationUrl URL where user can authenticate
```

Comment (jianghaolu) [reply sdk/identity/azure-identity/src/main/java/com/azure/identity/DeviceCodeChallenge.java]: Fixed here and everywhere

**Pull request merged**